blog/2020-10-23-supervisor-update.md

```diff
@@ -0,0 +1,90 @@
+---
+author: Joakim Sørensen
+authorURL: https://github.com/ludeeus
+authorTwitter: ludeeus
+title: "Calling all add-on developers!"
+---
+
+## Documentation
+
+First off, our [API documentation][api_docs] has been moved to the developer documentation site, during this move it also got a style update to make it easier to navigate. Some of the endpoints are still missing some content, if you have not yet met your quota for [Hacktoberfest] maybe you want to contribute some more details to our API descriptions?
+
+## API Changes
+
+- Using the `/homeassistant/*` endpoints is deprecated, and will be removed later this year. You need to use `/core/*` instead.
+- Using `http://hassio/` is deprecated, and will be removed later this year. You need to use `http://supervisor/` instead.
+- Using `HASSIO_TOKEN` is deprecated, and will be removed later this year. You need to use `SUPERVISOR_TOKEN` instead.
+- Deleting snapshots with `POST` calling `/supervisor/snapshots/<slug>/remove` is deprecated, and will be removed later this year. You need to use the `DELETE` method when calling `/supervisor/snapshots/<slug>` instead.
+- Using `X-Hassio-Key` header as authentication method is deprecated, and will be removed later this year. You need use an authorization header with a `Bearer` token instead.
+
+The [API documentation][api_docs] have been updated to reflect these changes.
+
+## Add-on options
+
+The permissions of the `/data/options.json` file, was change from `644` to `600`. If your add-on is running as non-root and you are reading this file, it will give now you permission issues.
```

Comment:
```suggestion
The permissions of the `/data/options.json` file, were changed from `644` to `600`. If your add-on is running as non-root and you are reading this file, it will now give you permission issues.
```